### ISSUE TITLE
Package removed from CRAN

### ISSUE THREAD

**Issue by danielnachun** [conda-forge/r-xroi-feedstock#2]
This package has been removed from CRAN as of 2023-10-16, so this feedstock needs to be archived.

**Comment by bgruening** [comment]
Are there any dependent packages of this one here?

**Comment by danielnachun** [comment]
No, only one package in `bioconda` - `bioconductor-spiat`. And as of the latest version, the upstream package no longer depends on it: https://www.bioconductor.org/packages/release/bioc/html/SPIAT.html.
